apps/web/src/components/transactions/TxShareLink/TxShareBlock.tsx

```diff
@@ -0,0 +1,37 @@
+import { Accordion, AccordionDetails, AccordionSummary, Button, Paper, SvgIcon, Typography } from '@mui/material'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import type { ReactElement } from 'react'
+
+import ShareIcon from '@/public/images/messages/link.svg'
+import { TX_LIST_EVENTS } from '@/services/analytics'
+import TxShareLink from '.'
+
+import css from './styles.module.css'
+
+export function TxShareBlock({ txId }: { txId: string }): ReactElement | null {
+  return (
+    <Paper className={css.wrapper}>
+      <Accordion className={css.accordion}>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />} className={css.summary}>
+          <Typography className={css.header}>Share link with other owners</Typography>
```

Comment:
```suggestion
          <Typography className={css.header}>Share link with other signers</Typography>
```

We've been calling them signers throughout the UI.

Reply:
Updated in fe946dd8909741ccba923ac5b402c9b99ebb8249.